src/common/components/layouts/pages/inspect/pdfDocument/PdfDocument.js

```diff
@@ -15,6 +15,10 @@ import Alert from '@material-ui/lab/Alert';
 import Close from '@material-ui/icons/Close';
 
 import './PdfDocument.scss';
+import { getItem } from '../../../../../services/localStorageService';
```

Comment:
Move these 4 new imports higher. Please them after the `import { getFileLinkById } from '../../../../../services/fileService';`.
We keep the following order of the imports:
```
imports from external libs;

imports from local modules;

imports from static files;
```